cmd/ttn-lw-cli/commands/contact_info.go

```diff
@@ -173,8 +174,36 @@ func contactInfoCommands(entity string, getID func(cmd *cobra.Command, args []st
 			return io.Write(os.Stdout, config.OutputFormat, updatedInfo)
 		},
 	}
+	validate := &cobra.Command{
+		Use:   "validate [reference] [confirmation-token]",
```

Comment:
If you use `confirmation-token` here, the flag should also be called `confirmation-token`.

Reply:
Also, you're not actually checking the args in RunE below

Reply:
I would just call it `token`; the context is clear enough here